build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# ****************************************************************************
+# Copyright 2004-2013 Castle Project - http://www.castleproject.org/
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ****************************************************************************
+
+if [ ! -d "./.dotnet" ]; then
+
+    sudo apt-get -qq update
+
+    sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+
+    sudo apt-get install -y libunwind8
+
+    wget -O - https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh | bash -s -- -i ./.dotnet/
+
+fi
+
+#RUNTIME="ubuntu.12.04-x64"
+#RUNTIME="ubuntu.16.04-x64"
+#RUNTIMEOPTS=" --runtime $RUNTIME"
+RUNTIMEOPTS=""
```

Comment:
This is me mucking around with RuntimeIdentifiers. I have a separate branch where I am using this to try and gets to install/build using the VS2017 tooling on Linux. So it might come back but it is not needed for now. Will remove it.